server/storage/schema/schema.go

```diff
@@ -131,8 +131,19 @@ var (
 		version.V3_6: {
 			addNewField(Meta, MetaStorageVersionName, emptyStorageVersion),
 		},
+		version.V3_5: {
+			// 'migrate' command from v3.4 to v3.5 isn't supported and will fail with `missing confstate information`
+			// these fields are added for v3.5 -> v3.4 downgrade only
+			//
+			// note about emptyValue:
+			// even if these fields are used for upgrade, it's safe to set emptyValue
+			// UnsafeReadConsistentIndex and UnsafeConfStateFromBackend checks for zero-length values
+			addNewField(Meta, MetaTermKeyName, emptyValue),
+			addNewField(Meta, MetaConfStateName, emptyValue),
```

Comment:
Also add `ClusterClusterVersionKeyName` and `ClusterDowngradeKeyName`?  https://github.com/etcd-io/etcd/blob/8cb3fab8d9ec6d4a395d0f46d85049b25cb90938/server/storage/schema/bucket.go#L78-L81

Reply:
Those fields were added in v3.4 not v3.5